LandSandBoat/server
# [WS] Add fTP to physical utilities and table elemental Belts and Gorgets

**_I affirm:_**

## What does this pull request do?

- Codes fTP to physical_utilities file.
- Adds elemental Belts and Gorgets (Including Fotia) to items.lua file.
- Corrects Belts and Gorgets table order, to match their actual element.

The function isnt called yet, so it does nothing ATM.
The change in weaponskills.lua fixes belts and Gorgets giving fTP bonuses to incorrect elements.. 

## Steps to test these changes

Do WSs with known elemental properties while wearing an apropiate belt and/or gorget.
Observe an slight damage increase. Or better yet, add debug prints to see if fTP gets a bonus or not.
